org.opentosca.container.util/pom.xml

```diff
@@ -0,0 +1,19 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">

```

Comment:
Why do we need an additional bundle? What's the problem with adding such generic stuff to the "core" package, on which basically everything is depending on?

Reply:
Anyhow, this `pom.xml` doesn't seem to work with our Tycho build structure.

Reply:
In case the NCName class will be merged (which is IMO the most likely case), all other classes/files you mentioned will be discarded. In this case the additional bundle would be obsolete as well, we simply have to find a new 'home' for this single file `NCName`. If everyone agrees with this opinion, the only change to make is removing all obsolete files and moving the NCName class into another package. This seems the easiest way to go in order to (finally) be able to fix issue #61 properly.